### ISSUE TITLE
Web Accessibility

### ISSUE THREAD

**Issue by James-Byrne** [brainsum/cookieconsent#77]
Hi there 👋 ,

We are looking at using this package in work but we also fall under the remit of [WCAG 2.1](https://www.w3.org/TR/WCAG21/) and need to maintain an accessible site. At the moment this package appears to be inaccessible by anyone who isn't a sighted user with a mouse (tested with MacOS voiceover) due to a lack of either semantic HTML or `aria-*` attributes.

Would you be open to pull requests that address these issues?
